priv/cabbage/apps/itest/lib/account.ex

```diff
@@ -24,7 +24,7 @@ defmodule Itest.Account do
   def take_accounts(number_of_accounts) do
     1..number_of_accounts
     |> Task.async_stream(fn _ -> account() end,
-      timeout: 60_000,
+      timeout: 240_000,
```

Comment:
Just for my knowledge, why the increase?

Reply:
the task is timing out https://app.circleci.com/pipelines/github/omgnetwork/elixir-omg/7388/workflows/229cd6c9-8ddf-4d0b-ba76-dd5827bdc467/jobs/66078/steps